spec/requests/alert_definitions_spec.rb

```diff
@@ -91,6 +91,66 @@
     )
   end
 
+  it "creates an alert definition with miq_expression" do
+    sample_alert_definition = {
+      "description"    => "Test Alert Definition",
+      "db"             => "ContainerNode",
+      "miq_expression" => { "eval_method" => "dwh_generic", "mode" => "internal", "options" => {} },
```

Comment:
Is this supposed to be a hash expression standing in place of an miq expression? Even if this "works", I wouldn't want us to officially support this surprising behavior